torch/_inductor/kernel/flex_attention.py

```diff
@@ -528,18 +640,29 @@ def flex_attention(*args, **kwargs):
             # For now, we always assume the "sound" option
             ROWS_GUARANTEED_SAFE=False,
             PRESCALE_QK=False,
+            USE_PARTIAL_MASK=(sum(partial_kv_num_blocks.shape) != 3),
```

Comment:
I think we have to figure out a better way to tell the Triton kernel if using partial mask, because I can't get if ```mask_fn``` is None from ```block_mask``` at here, probably we should add another flag.

Reply:
I decided still to check if ```partial_kv_indices``` is ```torch.zeros([1,1,1])``` to mark if need partial masking inside of triton kernel.